ignite/cmd/cmd.go

```diff
@@ -259,14 +259,7 @@ func checkNewVersion(ctx context.Context) {
 		return
 	}
 
-	fmt.Printf(`·
-· 🛸 Ignite CLI %s is available!
-·
-· To upgrade your Ignite CLI version, see the upgrade doc: https://docs.ignite.com/welcome/install#upgrading-your-ignite-cli-installation
-·
-··
-
-`, next)
+	fmt.Printf("⬆️ Ignite CLI %s is available! To upgrade: https://docs.ignite.com/welcome/install#upgrading-your-ignite-cli-installation", next)
```

Comment:
Guess you missed change this
```suggestion
	fmt.Printf("⬆️ Ignite CLI %s is available! To upgrade: https://docs.ignite.com/welcome/install#upgrade", next)
```